cmd/root.go

```diff
@@ -3,12 +3,13 @@ package cmd
 import (
 	_ "embed"
 	"fmt"
-	"github.com/sirupsen/logrus"
 	"os"
 	"os/signal"
 	"strings"
 	"syscall"
 
+	"github.com/sirupsen/logrus"
```

Comment:
why do we need logrus? we use zap

Reply:
https://github.com/project-n-oss/sidekick/blob/main/cmd/root.go#L111 this is a line added by you - is this an error and should it be `rootLogger.Fatal("failed to execute command", zap.Error(err))`

Reply:
Ahhhh the classic it was me all along.

I fixed the issue and pushed a commit with a one-line change. You can revert if you don't like it